### PR TITLE
[WIP] Add docs for KEP-1287 In-place Update of Pod Resources

### PR DESCRIPTION
**Pull request by vinaykul**
Issue link: https://github.com/kubernetes/enhancements/issues/1287
PR link: https://github.com/kubernetes/kubernetes/pull/102884

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [WIP] Add docs for KEP-1287 In-place Update of Pod Resources

File: content/en/docs/tasks/configure-pod-container/resize-container-resources.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
+---
+title: Resize CPU and Memory Resources assigned to Containers of a Pod
+content_template: templates/task
+weight: 20
+---
+
+{{% capture overview %}}
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/resize-container-resources.md:7]: @vinaykul it looks like you are using a page template from a few years ago. Check other pages within `content/en/docs/tasks/` for an example of what a task page looks like now.

**Review by sftim — COMMENTED** [review]
Some early feedback.

**Comment by reylejano** [comment]
/milestone 1.25
/assign @sethmccombs 
/cc @kcmartin 
/hold
I placed a hold because the related enhancement did not make the 1.25 code freeze, see:
https://github.com/kubernetes/enhancements/issues/1287

File: content/en/docs/tasks/configure-pod-container/resize-container-resources.md (hunk 1 above)
Comment (vinaykul) [reply content/en/docs/tasks/configure-pod-container/resize-container-resources.md:7]: Thanks for the feedback Tim. Yes this feature has gone through a few redesigns and I had written up the documentation when the earlier design was near implemented and we decided that it needed to be reworked. I'll update to use the latest template in the next milestone.

**Comment by vinaykul** [comment]
> /milestone 1.25 /assign @sethmccombs /cc @kcmartin /hold I placed a hold because the related enhancement did not make the 1.25 code freeze, see: [kubernetes/enhancements#1287](https://github.com/kubernetes/enhancements/issues/1287)

/milestone clear

@reylejano Yes. There were some late changes (GKE switching to cgroup v2 in CI when I had planned adding support for it pre-beta) that broke my E2E test and I was not very comfortable with the late game code churn to handle it. We extracted the near-zero risk CRI portion of the code for this milestone, with plans to bring in containerd CRI support as quickly as possible and then bring in the rest of this feature in 1.26.

I'll fix and track this PR to dev-1.26 when that is ready to take changes.

**Comment by reylejano** [comment]
@vinaykul , for now can you change the target branch to `main` 
[Here's an example from GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)

The `dev-1.26` branch will be created shortly after 1.25 is released
/milestone clear

**Comment by jimangel** [comment]
Hey @vinaykul, with the `dev-1.25` branch merged, and after a few weeks of no changes, I'm going to close this PR. This is more for keeping our inactive PR volume low, you're very welcome (and encouraged) to open it up again using the `dev-1.26` branch when time permits! Thanks!

/close

**Pull request closed without merging**